sms.info.yml

```diff
@@ -5,4 +5,6 @@ core: 8.x
 type: module
 project: smsframework
 configure: admin/config/smsframework
-
+dependencies:
+  - telephone
+  - dynamic_entity_reference
```

Comment:
I'm a bit concerned about this dependency, but I guess since DER is lined up for core in 8.1.x or 8.2.x, there isn't too much worries.